tests/system/feature_store/test_feature_store.py

```diff
@@ -334,7 +334,7 @@ def test_ingest_partitioned_by_key_and_time(
 
         file_system = fsspec.filesystem("v3io")
         dataset = pq.ParquetDataset(
-            f"v3io:///projects/system-test-project/fs/parquet/sets/{name}-latest",
+            f"v3io:///projects/system-test-project/FeatureStore/{name}/parquet/sets/{name}-latest",
```

Comment:
Better to change it here to use `get_default_prefix_for_target` so the test won't brake again when changing the default